purchases/src/main/kotlin/com/revenuecat/purchases/models/TestStoreProduct.kt

```diff
@@ -0,0 +1,95 @@
+package com.revenuecat.purchases.models
+
+import com.revenuecat.purchases.ProductType
+
+/**
+ * A test-only [StoreProduct] implementation.
+ * This can be used to create mock data for tests or Jetpack Compose previews.
+ */
+data class TestStoreProduct(
+    override val id: String,
+    override val title: String,
+    override val description: String,
+    override val price: Price,
+    override val period: Period?,
+    private val freeTrialPeriod: Period? = null,
+    private val introPrice: Price? = null,
```

Comment:
I guess these 2 fields would only apply to subscriptions, so I wonder if we should make the API receive a sealed class for Subscription/InApp to support these 2... Anyway I think it's ok for now

Reply:
That actually makes sense.

Since I'm merging this in `paywalls` we have time to improve this.